docs/docs/other-css-frameworks.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Other CSS Frameworks
+---
+
+This guide introduces some other popular styling libraries that you might want to use in your project.
+
+## How these frameworks were chosen
+
+The following criteria were used to determine which frameworks to include in this guide:
+
+- It must work with React and Gatsby.
+- It must impact the styling of a Gatsby site.
+- It should be a commonly requested or searched in the Gatsby docs.
+- It must consider accessibility and not promote UI anti-patterns.
+
+## Bootstrap
+
+Bootstrap is a widely used CSS and JS framework that was created at Twitter in 2010. It gained popularity for its support of responsive and mobile-first design.
+
+Under the hood, Bootstrap uses [Sass](https://sass-lang.com/). Unlike other frameworks in this guide, Bootstrap does not provide React components directly. Instead, it provides a set of pre-built CSS class names like `"btn-primary"` or `"dropdown"` that you can use to style HTML elements.
+
+If you're using Bootstrap v4 or older, many of the JavaScript components will need you to also install jQuery to work correctly. As of [Bootstrap v5](https://blog.getbootstrap.com/2020/06/16/bootstrap-5-alpha/), the framework no longer depends on jQuery.
+
+**Browser-support warning:** Bootstrap v5 also dropped Internet Explorer support.
+
+If you're interested in using Bootstrap and want to host the CSS and JS files yourself, you can install the [Bootstrap npm package](https://www.npmjs.com/package/bootstrap). Or, if you'd rather not host the files yourself, you can source them using the [Bootstrap Content Delivery Network (CDN)](https://www.bootstrapcdn.com/).
+
+- [Bootstrap documentation](https://getbootstrap.com/docs)
+- [Bootstrap repo on GitHub](https://github.com/twbs/bootstrap)
+
+## Chakra UI
+
+Chakra UI is built on top of the [Emotion styling library](https://emotion.sh/docs/introduction). It uses [Styled System](https://styled-system.com/) style props to override individual CSS properties for a component. Chakra UI supports theme customization, provides a library of components, and comes with dark mode out of the box.
```

Comment:
You may want to specify exactly what you mean by "theme" here. I could see folks getting confused because of the existence of Gatsby themes.